app/app/src/main/res/layout/activity_second.xml

```diff
@@ -0,0 +1,25 @@
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/black"
```

Comment:
This conflicts with issue #78 . Colour elements in the ui should match the light mode in case we want to implement a dark mode later on.